README.md

```diff
@@ -1,163 +1,257 @@
 ```
 
-Create a new pipeline at launchpad
-_NOTE: the params option has to be a file. Here we are using Bash pipes to convert a command output into a file automatically._
+*The `params` option should be a YAML or JSON file. Here we use a Bash pipe to convert a command into a YAML file automatically.*
+
+Pipelines consists of a workflow repository, launch parameters, and a Compute Environment. When no Compute Environment is specified, the primary CE is used.
+
+
+### 9. Launch it! :rocket:
+
+Run the workflow.
 
 ```
-tw pipelines create -n sleep_one_minute --params=<(echo 'timeout: 60') https://github.com/pditommaso/nf-sleep
+$ tw launch sleepy-flow
 ```
 
-Run it!
+### 10. Change launch parameters
+
+Launch the workflow with different parameters.
 
 ```
-tw launch sleep_one_minute
+$ tw launch sleepy-flow --params=<(echo 'timeout: 30')
 ```
 
-Run it with different parameters
+### 11. Update a Pipeline
+
+Pipelines can be modified with new launch parameters using the `update` command. 
 
 ```
-tw launch sleep_one_minute --params=<(echo 'timeout: 30')
+$ tw pipelines update -n sleepy-flow --params=<(echo 'timeout: 30')
 ```
 
-Directly launch a Github pipeline not defined at launchpad
+### 12. Launch a workflow directly
+
+It possible to directly launch Git workflows that have not been added to a workspace as Pipelines.
 
 ```
-tw launch nextflow-io/hello
+$ tw launch nextflow-io/hello
 ```
 
-Update a launchpad pipeline
+### :gift: Bonus: Activate autocomplete 
+
+Install autocomplete in your current session, type `tw pi` then press tab twice.
 
 ```
-tw pipelines update -n sleep_one_minute --params=<(echo 'timeout: 30')
+$source <(tw generate-completion)
 ```
 
-## Launch usage examples
+Launch Examples
+---------------
 
-Run a workspace defined pipeline with custom parameters file:
+The `tw launch` command provides a similar user experience to `nextflow run` with the benefits of using Tower.
+
+### 1. Run a workspace Pipeline with a custom parameters file
 
 ```
 tw launch sarek --params ./myparams.yaml
 ```
 
-Run any Nextflow pipeline using the primary compute environment:
+### 2. Run any Nextflow workflow using the primary Compute Environment
 
 ```
-tw launch nextflow-io/hello 
+$ tw launch nextflow-io/hello 
 ```
 
-Run any Nextflow pipeline setting a profile.
+### 3. Run any Nextflow pipeline setting a profile
 
 ```
-tw launch nf-core/sarek --profile test,docker --params ./myparams.yaml
+$ tw launch nf-core/sarek --profile test,docker --params ./myparams.yaml
 ```
 
-Select the compute environment that you want to use:
+### 4. Select a specific Compute Environment
 
 ```
-tw launch nf-core/sarek --compute-env "aws seqera" --profile test,docker
+$ tw launch nf-core/sarek --compute-env "aws-ce" --profile test,docker
 ```
 
-### Development
+
+Build binary development version
+--------------------------------
+
+The Tower CLI is based on a Java native compilation, Micronaut HTTP declarative client and Tower domain classes.
```

Comment:
We need to update this sentence to something like: "Tower CLI is based on a Java GraalVM native compilation to produce a platform binary executable."